raiden/transfer/channel.py

```diff
@@ -1743,7 +1780,14 @@ def send_expired_withdraws(
             break
 
         nonce = get_next_nonce(channel_state.our_state)
-        channel_state.our_state.nonce = nonce
+        coop_settle = channel_state.our_state.initiated_coop_settle
+        if coop_settle is not None:
+            if (
+                coop_settle.total_withdraw_participant == withdraw_state.total_withdraw
+                and coop_settle.expiration == withdraw_state.expiration
+            ):
+                # We also declare the coop-settle as expired, when we initated it
```

Comment:
initated -> initiated

Reply:
And I think it should be `if` instead of `when`.

Reply:
Done.